Cloud Application Manager/Automating Deployments/script-box.md

```diff
@@ -9,12 +9,13 @@
 
```

Comment:
Please, update date and include you in author (you can keep previous by adding: `and <your_name>`)